java/memory/memory-core/src/test/java/org/apache/arrow/memory/TestAccountant.java

```diff
@@ -16,10 +16,11 @@
  */
 package org.apache.arrow.memory;
 
-import static org.junit.Assert.assertEquals;
+import org.junit.jupiter.api.Test;
 
-import org.junit.Assert;
-import org.junit.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.fail;
```

Comment:
For consistency, it would be best to place `import static` statements before regular `import` statements.

Reply:
I looked at some of the other commits. Do you want me to keep the 'public' modifier on the tests that have been updated to JUnit 5 for consistency as well?

Reply:
Good point.

I also noticed that the accessor is ambiguous. However, the main goal of these PRs is to change the JUnit version from 4 to 5, so I did not modify the accessor to maintain consistency.